app/scripts/modules/core/src/pipeline/config/stages/evaluateVariables/EvaluateVariablesStageConfig.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import { map, set } from 'lodash';
+
+import { IStageConfigProps, StageConfigField, MapEditor } from '@spinnaker/core';
```

Comment:
When in `core`, don't import from `@spinnaker/core`

Reply:
Thanks! I was just looking at that.